public/editor/scripts/project-list.js

```diff
@@ -81,44 +81,88 @@ require(["jquery", "constants", "analytics", "moment"], function($, Constants, a
 
   $("#project-list").prepend(favoriteProjectsElementList);
 
+  var projectsToDelete = [];
+
   $(".project-delete").click(function() {
-    // TODO: we can do better than this, but let's at least make it harder to lose data.
-    if(!window.confirm("{{ deleteProjectConfirmationText }}")) {
-      return false;
+    if($(".delete-button").length === 0){
+      var deleteBtn = document.createElement("div");
+      deleteBtn.appendChild(document.createTextNode("Delete Projects"));
+      deleteBtn.className = "delete-button";
+      deleteBtn.onclick = function() {
```

Comment:
do not use `on...` handlers in modern JS. Instead use `addEventListener` in plain JS, but since we have jQuery, if we use the above-mentioned code we can instead do:
```
deleteButton.on("click", function(evt) { .... });
```